physionet-django/project/managers/project.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import Manager, Q
+from project.models import AccessPolicy, DUASignature, DataAccessRequest
+from user.models import Training, TrainingType
+
+
+class PublishedProjectManager(Manager):
+    def accessible_by(self, user):
```

Comment:
We need this logic in the environments app. Figured it may come useful somewhere else as well.